module_utils/network_lsr/argument_validator.py

```diff
@@ -1547,6 +1564,20 @@ def _validate_post(self, value, name, result):
                         "but is '%s'" % result["mac"],
                     )
 
+            if "match" in result:
+                if "path" in result["match"]:
+                    if result["match"]["path"] == []:
+                        raise ValidationError(
+                            name + "match.path",
+                            "'match.path' can not be empty list",
+                        )
```

Comment:
@thom311 , It seems that NM allow empty list for the `match.path` setting, should we remove the validation here ?